src/include/txn_inline.h

```diff
@@ -631,6 +631,57 @@ __wt_txn_oldest_id(WT_SESSION_IMPL *session)
     }
 }
 
+/*
+ * __wt_txn_pinned_stable_timestamp --
+ *     Get the first timestamp that can be written to the disk for precise checkpoint.
+ */
+static WT_INLINE void
+__wt_txn_pinned_stable_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t *pinned_stable_tsp)
```

Comment:
I don't see this used... I'm assuming you'd use this in the next ticket?

Reply:
Yes, that's correct